prow/tide/status_test.go

```diff
@@ -439,20 +438,3 @@ func TestTargetUrl(t *testing.T) {
 		}
 	}
 }
-
-func TestAllOpenPRs(t *testing.T) {
```

Comment:
Why the removal?

Reply:
I meant to move this not remove it. Fixed.